src/SearchComponent/SearchView.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react'
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+} from "react-router-dom";
+import '../App.css';
+import {connect} from 'react-redux'
+import { SearchActions } from '../Redux/SearchRedux'
+import MovieRowComponent from './MovieRowComponent'
+import SearchBar from './SearchBarComponent'
+import Paper from "@material-ui/core/Paper";
+import Grid from "@material-ui/core/Grid";
+
+// A class representing a component ("view") that displays the search screen.
+class SearchView extends Component {
+
+  // Returns the component responsible for displaying the search screen.
+  searchView() {
+    const { searchResults, onRequestMovies, error } = this.props
+
+    return (
+      <div>
+        <div className={{flexGrow: 1}}>
+          <Grid container spacing={2}>
+            <Grid item xs={9}>
+              <SearchBar
+                onEnterPress={onRequestMovies}
```

Comment:
`SearchBar` is not using a prop called `onEnterPress`. You can remove all places of `onRequestMovies` in this file.